### PR TITLE
Update license headers

### PR DESCRIPTION
**Pull request by DeepDiver1975**
@karlitschek updated license headers - as requested

### CODE REVIEW

File: apps/files_sharing/templates/list.php [hunk 1]
```diff
@@ -1,4 +1,24 @@
-<?php /** @var $l OC_L10N */ ?>
```
Comment (LukasReschke) [inline apps/files_sharing/templates/list.php]: :see_no_evil:

File: apps/files_sharing/templates/public.php [hunk 2]
```diff
@@ -1,7 +1,35 @@
 <?php
-/** @var $l OC_L10N */
-/** @var $_ array */
```
Comment (LukasReschke) [inline apps/files_sharing/templates/public.php]: :see_no_evil:

Comment (LukasReschke) [inline apps/files_sharing/templates/public.php]: and a lot of other occurences like that

File: apps/files_sharing/templates/list.php (hunk 1 above)
Comment (DeepDiver1975) [inline apps/files_sharing/templates/list.php]: autsch - :boom:

**Comment by DeepDiver1975** [comment]
:gun: :bomb: :boom: @owncloud-bot retest this please :boom: :bomb: :gun:

**Comment by DeepDiver1975** [comment]
:gun: :bomb: :boom: @owncloud-bot retest this please :boom: :bomb: :gun:

**Comment by karlitschek** [comment]
:+1: Can we also add the email addresses to the author field?
something like: @author 2015 Frank Karlitschek frank@owncloud.org

**Comment by DeepDiver1975** [comment]
This patch is AWSOME! Test coverage of 10000%


bildschirmfoto von 2015-02-20 16 28 45

**Comment by DeepDiver1975** [comment]
> :+1: Can we also add the email addresses to the author field?

let me see ....

**Comment by DeepDiver1975** [comment]
@karlitschek email has been added - we follow now the spec where no year is added https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#82-author

**Comment by DeepDiver1975** [comment]
:gun: :bomb: :boom: @owncloud-bot retest this please :boom: :bomb: :gun:

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please.

**Comment by karlitschek** [comment]
great :+1:

**Comment by Xenopathic** [comment]
I'm impressed with the script that managed to pull that off :+1:

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please

**Comment by DeepDiver1975** [comment]
unit test execution succeeds locally -> merge

```sh
deepdiver@alien:~/Development/ownCloud/master [update-license-headers]$ ./autotest.sh mysql
Using database oc_autotest
Setup environment for mysql testing ...
INDEX
END INDEX
Testing with mysql ...
PHPUnit 4.3.5 by Sebastian Bergmann.

Configuration read from /home/deepdiver/Development/ownCloud/master/tests/phpunit-autotest.xml

..............S..............................................   61 / 2980 (  2%)
.............................................................  122 / 2980 (  4%)
.............................................................  183 / 2980 (  6%)
.............................................................  244 / 2980 (  8%)
.............................................................  305 / 2980 ( 10%)
.............................................................  366 / 2980 ( 12%)
.............................................................  427 / 2980 ( 14%)
.............................................................  488 / 2980 ( 16%)
.............................................................  549 / 2980 ( 18%)
.............................................................  610 / 2980 ( 20%)
.............................................................  671 / 2980 ( 22%)
.......S.....................................................  732 / 2980 ( 24%)
.............................................................  793 / 2980 ( 26%)
.............................................................  854 / 2980 ( 28%)
..............................................SSSSSS.........  915 / 2980 ( 30%)
.............................................................  976 / 2980 ( 32%)
.............................SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS 1037 / 2980 ( 34%)
SSSSSSSSSSSSSSSSSSSSSSSSSS................................... 1098 / 2980 ( 36%)
............................................................. 1159 / 2980 ( 38%)
............................................................. 1220 / 2980 ( 40%)
............................................................. 1281 / 2980 ( 42%)
............................................................. 1342 / 2980 ( 45%)
............................................................. 1403 / 2980 ( 47%)
............................................................. 1464 / 2980 ( 49%)
............................................................. 1525 / 2980 ( 51%)
............................................................. 1586 / 2980 ( 53%)
............................................................. 1647 / 2980 ( 55%)
............................................................. 1708 / 2980 ( 57%)
............................................................. 1769 / 2980 ( 59%)
............................................................. 1830 / 2980 ( 61%)
............................................................. 1891 / 2980 ( 63%)
............................................................. 1952 / 2980 ( 65%)
.........................................SS...........SSSSSSS 2013 / 2980 ( 67%)
SSSSSSSSSSSSS..........SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS........ 2074 / 2980 ( 69%)
............................................................. 2135 / 2980 ( 71%)
............................................................. 2196 / 2980 ( 73%)
............................................................. 2257 / 2980 ( 75%)
............................................................. 2318 / 2980 ( 77%)
............................................................. 2379 / 2980 ( 79%)
............................................................. 2440 / 2980 ( 81%)
............................................................. 2501 / 2980 ( 83%)
......S..........................SS...........S.............. 2562 / 2980 ( 85%)
............................................................. 2623 / 2980 ( 88%)
............................................................. 2684 / 2980 ( 90%)
............................................................. 2745 / 2980 ( 92%)
............................................................. 2806 / 2980 ( 94%)
............................................................. 2867 / 2980 ( 96%)
............................................................. 2928 / 2980 ( 98%)
....................................................

Time: 41.04 minutes, Memory: 227.00Mb

There were 122 skipped tests:

1) Test\TestAllConfig::testSetUserValueUnchanged
Skipped because this is just testable if database connection can be injected

/home/deepdiver/Development/ownCloud/master/tests/lib/allconfig.php:170

2) TestSqliteMigration::testNonOCTables
Test only relevant on Sqlite

/home/deepdiver/Development/ownCloud/master/tests/lib/db/sqlitemigration.php:22

3) Test\Files\Filesystem::testNormalizePathWindowsAbsolutePath with data set #0 ('C:/', 'C:\\')
This test is Windows only

/home/deepdiver/Development/ownCloud/master/tests/lib/files/filesystem.php:228

4) Test\Files\Filesystem::testNormalizePathWindowsAbsolutePath with data set #1 ('C:/', 'C:\\', false)
This test is Windows only

/home/deepdiver/Development/ownCloud/master/tests/lib/files/filesystem.php:228

5) Test\Files\Filesystem::testNormalizePathWindowsAbsolutePath with data set #2 ('C:/tests', 'C:\\tests')
This test is Windows only

/home/deepdiver/Development/ownCloud/master/tests/lib/files/filesystem.php:228

6) Test\Files\Filesystem::testNormalizePathWindowsAbsolutePath with data set #3 ('C:/tests', 'C:\\tests', false)
This test is Windows only

/home/deepdiver/Development/ownCloud/master/tests/lib/files/filesystem.php:228

7) Test\Files\Filesystem::testNormalizePathWindowsAbsolutePath with data set #4 ('C:/tests', 'C:\\tests\\')
This test is Windows only

/home/deepdiver/Development/ownCloud/master/tests/lib/files/filesystem.php:228

8) Test\Files\Filesystem::testNormalizePathWindowsAbsolutePath with data set #5 ('C:/tests/', 'C:\\tests\\', false)
This test is Windows only

/home/deepdiver/Development/ownCloud/master/tests/lib/files/filesystem.php:228

9) OCA\ObjectStore\Tests\Unit\Swift::testStat
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

10) OCA\ObjectStore\Tests\Unit\Swift::testMove with data set #0 ('/source.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

11) OCA\ObjectStore\Tests\Unit\Swift::testMove with data set #1 ('/source.txt', '/target with space.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

12) OCA\ObjectStore\Tests\Unit\Swift::testMove with data set #2 ('/source with space.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

13) OCA\ObjectStore\Tests\Unit\Swift::testMove with data set #3 ('/source with space.txt', '/target with space.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

14) OCA\ObjectStore\Tests\Unit\Swift::testMove with data set #4 ('/source.txt', '/tärgét.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

15) OCA\ObjectStore\Tests\Unit\Swift::testMove with data set #5 ('/sòurcē.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

16) OCA\ObjectStore\Tests\Unit\Swift::testMove with data set #6 ('/sòurcē.txt', '/tärgét.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

17) OCA\ObjectStore\Tests\Unit\Swift::testRenameDirectory
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

18) OCA\ObjectStore\Tests\Unit\Swift::testRenameOverWriteDirectory
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

19) OCA\ObjectStore\Tests\Unit\Swift::testRenameOverWriteDirectoryOverFile
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

20) OCA\ObjectStore\Tests\Unit\Swift::testRoot
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

21) OCA\ObjectStore\Tests\Unit\Swift::testTestFunction
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

22) OCA\ObjectStore\Tests\Unit\Swift::testDirectories with data set #0 ('folder')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

23) OCA\ObjectStore\Tests\Unit\Swift::testDirectories with data set #1 (' folder')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

24) OCA\ObjectStore\Tests\Unit\Swift::testDirectories with data set #2 ('folder ')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

25) OCA\ObjectStore\Tests\Unit\Swift::testDirectories with data set #3 ('folder with space')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

26) OCA\ObjectStore\Tests\Unit\Swift::testDirectories with data set #4 ('spéciäl földer')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

27) OCA\ObjectStore\Tests\Unit\Swift::testGetPutContents with data set #0 ('/home/deepdiver/Development/ownCloud/master/tests/data/lorem.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

28) OCA\ObjectStore\Tests\Unit\Swift::testGetPutContents with data set #1 ('/home/deepdiver/Development/ownCloud/master/tests/data/lorem-big.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

29) OCA\ObjectStore\Tests\Unit\Swift::testMimeType
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

30) OCA\ObjectStore\Tests\Unit\Swift::testCopy with data set #0 ('/source.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

31) OCA\ObjectStore\Tests\Unit\Swift::testCopy with data set #1 ('/source.txt', '/target with space.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

32) OCA\ObjectStore\Tests\Unit\Swift::testCopy with data set #2 ('/source with space.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

33) OCA\ObjectStore\Tests\Unit\Swift::testCopy with data set #3 ('/source with space.txt', '/target with space.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

34) OCA\ObjectStore\Tests\Unit\Swift::testCopy with data set #4 ('/source.txt', '/tärgét.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

35) OCA\ObjectStore\Tests\Unit\Swift::testCopy with data set #5 ('/sòurcē.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

36) OCA\ObjectStore\Tests\Unit\Swift::testCopy with data set #6 ('/sòurcē.txt', '/tärgét.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

37) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverwrite with data set #0 ('/source.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

38) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverwrite with data set #1 ('/source.txt', '/target with space.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

39) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverwrite with data set #2 ('/source with space.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

40) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverwrite with data set #3 ('/source with space.txt', '/target with space.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

41) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverwrite with data set #4 ('/source.txt', '/tärgét.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

42) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverwrite with data set #5 ('/sòurcē.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

43) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverwrite with data set #6 ('/sòurcē.txt', '/tärgét.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

44) OCA\ObjectStore\Tests\Unit\Swift::testMoveOverwrite with data set #0 ('/source.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

45) OCA\ObjectStore\Tests\Unit\Swift::testMoveOverwrite with data set #1 ('/source.txt', '/target with space.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

46) OCA\ObjectStore\Tests\Unit\Swift::testMoveOverwrite with data set #2 ('/source with space.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

47) OCA\ObjectStore\Tests\Unit\Swift::testMoveOverwrite with data set #3 ('/source with space.txt', '/target with space.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

48) OCA\ObjectStore\Tests\Unit\Swift::testMoveOverwrite with data set #4 ('/source.txt', '/tärgét.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

49) OCA\ObjectStore\Tests\Unit\Swift::testMoveOverwrite with data set #5 ('/sòurcē.txt', '/target.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

50) OCA\ObjectStore\Tests\Unit\Swift::testMoveOverwrite with data set #6 ('/sòurcē.txt', '/tärgét.txt')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

51) OCA\ObjectStore\Tests\Unit\Swift::testLocal
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

52) OCA\ObjectStore\Tests\Unit\Swift::testUnlink
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

53) OCA\ObjectStore\Tests\Unit\Swift::testFOpen
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

54) OCA\ObjectStore\Tests\Unit\Swift::testTouchCreateFile
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

55) OCA\ObjectStore\Tests\Unit\Swift::testRecursiveRmdir
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

56) OCA\ObjectStore\Tests\Unit\Swift::testRecursiveUnlink
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

57) OCA\ObjectStore\Tests\Unit\Swift::testHash with data set #0 ('Foobar', 'md5')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

58) OCA\ObjectStore\Tests\Unit\Swift::testHash with data set #1 ('Foobar', 'sha1')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

59) OCA\ObjectStore\Tests\Unit\Swift::testHash with data set #2 ('Foobar', 'sha256')
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

60) OCA\ObjectStore\Tests\Unit\Swift::testHashInFileName
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

61) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverWriteFile
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

62) OCA\ObjectStore\Tests\Unit\Swift::testRenameOverWriteFile
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

63) OCA\ObjectStore\Tests\Unit\Swift::testCopyDirectory
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

64) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverWriteDirectory
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

65) OCA\ObjectStore\Tests\Unit\Swift::testCopyOverWriteDirectoryOverFile
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

66) OCA\ObjectStore\Tests\Unit\Swift::testInstanceOfStorage
objectstore tests are unreliable in some environments

/home/deepdiver/Development/ownCloud/master/tests/lib/files/objectstore/swift.php:37

67) Test\LargeFileHelperGetFileSize::testGetFileSizeViaCOM with data set #0 ('/home/deepdiver/Development/ownCloud/master/tests/data/lorem.txt', 446)
The PHP Windows COM extension is required for this test.

/home/deepdiver/Development/ownCloud/master/tests/lib/largefilehelpergetfilesize.php:66

68) Test\LargeFileHelperGetFileSize::testGetFileSizeViaCOM with data set #1 ('/home/deepdiver/Development/ownCloud/master/tests/data/strängé filename (duplicate #2).txt', 446)
The PHP Windows COM extension is required for this test.

/home/deepdiver/Development/ownCloud/master/tests/lib/largefilehelpergetfilesize.php:66

69) Test\Memcache\APC::testExistsAfterSet
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

70) Test\Memcache\APC::testGetAfterSet
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

71) Test\Memcache\APC::testGetArrayAfterSet
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

72) Test\Memcache\APC::testDoesNotExistAfterRemove
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

73) Test\Memcache\APC::testArrayAccessSet
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

74) Test\Memcache\APC::testArrayAccessGet
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

75) Test\Memcache\APC::testArrayAccessExists
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

76) Test\Memcache\APC::testArrayAccessUnset
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

77) Test\Memcache\APC::testSimple
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

78) Test\Memcache\APC::testClear
The apc extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apc.php:17

79) Test\Memcache\APCu::testExistsAfterSet
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

80) Test\Memcache\APCu::testGetAfterSet
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

81) Test\Memcache\APCu::testGetArrayAfterSet
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

82) Test\Memcache\APCu::testDoesNotExistAfterRemove
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

83) Test\Memcache\APCu::testArrayAccessSet
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

84) Test\Memcache\APCu::testArrayAccessGet
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

85) Test\Memcache\APCu::testArrayAccessExists
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

86) Test\Memcache\APCu::testArrayAccessUnset
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

87) Test\Memcache\APCu::testSimple
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

88) Test\Memcache\APCu::testClear
The APCu extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/apcu.php:17

89) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

90) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

91) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

92) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

93) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

94) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

95) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

96) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

97) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

98) Test\Memcache\Memcached
The memcached extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/memcached.php:17

99) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

100) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

101) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

102) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

103) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

104) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

105) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

106) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

107) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

108) Test\Memcache\Redis
The redis extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/redis.php:17

109) Test\Memcache\XCache::testExistsAfterSet
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

110) Test\Memcache\XCache::testGetAfterSet
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

111) Test\Memcache\XCache::testGetArrayAfterSet
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

112) Test\Memcache\XCache::testDoesNotExistAfterRemove
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

113) Test\Memcache\XCache::testArrayAccessSet
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

114) Test\Memcache\XCache::testArrayAccessGet
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

115) Test\Memcache\XCache::testArrayAccessExists
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

116) Test\Memcache\XCache::testArrayAccessUnset
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

117) Test\Memcache\XCache::testSimple
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

118) Test\Memcache\XCache::testClear
The xcache extension is not available.

/home/deepdiver/Development/ownCloud/master/tests/lib/memcache/xcache.php:17

119) OC\Settings\Controller\MailSettingsControllerTest::testSendTestMail
Disable test until OC_Mail is rewritten.

/home/deepdiver/Development/ownCloud/master/tests/settings/controller/mailsettingscontrollertest.php:159

120) OC\Settings\Controller\UsersControllerTest::testCreateUnsuccessfulWithInvalidEmailAdmin
Disable test until OC_Mail is rewritten.

/home/deepdiver/Development/ownCloud/master/tests/settings/controller/userscontrollertest.php:1158

121) OC\Settings\Controller\UsersControllerTest::testCreateSuccessfulWithValidEmailAdmin
Disable test until OC_Mail is rewritten.

/home/deepdiver/Development/ownCloud/master/tests/settings/controller/userscontrollertest.php:1185

122) OC\Core\LostPassword\Controller\LostControllerTest::testEmailSuccessful
sendmail is not available

/home/deepdiver/Development/ownCloud/master/tests/core/lostpassword/controller/lostcontrollertest.php:111

OK, but incomplete, skipped, or risky tests!
Tests: 3160, Assertions: 9636, Skipped: 122.

**Comment by MorrisJobke** [comment]
Yay \o/ dropped documentation of the class if there is no namespace for a class:

Before:
https://github.com/owncloud/core/blob/b057ae2da13964bb79e51286f8a34ae595ee4a2f/lib/private/legacy/appconfig.php#L30

After: 
https://github.com/owncloud/core/blob/8a4fddeb4afef2e96cadd505600700202fc0ee9b/lib/private/legacy/appconfig.php#L25

And it also dropped `@deprecated` statements in classes.

**Comment by MorrisJobke** [comment]
Volunteers? :D

**Comment by MorrisJobke** [comment]
Can we please revert this? Otherwise it will be much fun to dig through all the changed files and check for dropped documentation of the old classes without namespace. (`OC_` classes and legacy stuff)

**Comment by MorrisJobke** [comment]
Noticed because this caused a merge conflict in there: https://github.com/owncloud/core/pull/13938

**Comment by LukasReschke** [comment]
:+1: for reverting and fixing the script, seems like the easier option.

**Comment by MorrisJobke** [comment]
> Sorry, this pull request couldn't be reverted automatically. It may have already been reverted, or the content may have changed since it was merged.

Very funny. Much work. So wow.

**Comment by DeepDiver1975** [comment]
> :+1: for reverting and fixing the script, seems like the easier option.

yes - :disappointed: - I'll fix the script .... who can take care about the revert?

**Comment by MorrisJobke** [comment]
> who can take care about the revert?

I can give it a try.

**Comment by MorrisJobke** [comment]
Just 22 conflicts

**Comment by DeepDiver1975** [comment]
> Just 22 conflicts

can you resolve them? THX

**Comment by MorrisJobke** [comment]
I'm on it ;)

**Comment by MorrisJobke** [comment]
#14530

**Pull request merged**